lib/graphql/plug/endpoint.ex

```diff
@@ -8,8 +8,7 @@ defmodule GraphQL.Plug.Endpoint do
     opts
   end
 
-  def call(%Conn{method: req_method, params: %{"query" => query}} = conn, opts)
-  when req_method in ["GET", "POST"] do
+  def call(%Conn{method: req_method, params: %{"query" => query}} = conn, opts) when req_method in ["GET", "POST"] do
```

Comment:
@joshprice Fair enough, I’ve backed that change out for now. Having done a quick search it seems the answer is “no there isn’t” to the convention question. I don’t love the long lines either, but broken up I actually completely missed those guard clauses initially.